File: 240426/높은 수에서 낮은 수까지/from-high-to-low.js
const fs = require('fs');
const [a, b] = fs.readFileSync(0).toString().split(" ").map(Number);

const max = Math.max(a, b);
const min = Math.min(a, b);

for(let i = max; i >= min; i--){
    process.stdout.write(i + " ");
}

File: 240426/높은 수에서 낮은 수까지/from-high-to-low.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();

        int max = Math.max(a, b);
        int min = Math.min(a, b);

        for(int i = max; i >= min; i--){
            System.out.print(i + " ");
        }

    }
}